### PR TITLE
Add Slack team information to OrganizationSlackInfo

### PR DESCRIPTION
**Pull request by leogrim**
@andrewconner

### CODE REVIEW

**Comment by andrewconner** [comment]
:+1:

**Comment by ryanpbrewster** [comment]
My condolences for the merge conflicts :(

File: kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala [hunk 1]
```diff
@@ -214,21 +219,22 @@ class SlackInfoCommanderImpl @Inject() (
         }.toMap
       }
       val integrationInfosByLib = generateLibrarySlackIntegrationInfos(viewerId, slackToLibs, libToSlacks)
-      val slackTeams = slackTeamRepo.getByOrganizationId(orgId).map(_.slackTeamId)
+      val slackTeams = slackTeamRepo.getByOrganizationId(orgId).map(team => OrganizationSlackTeamInfo(team.slackTeamId, team.slackTeamName, team.lastChannelCreatedAt.map(_.toDateTime)))
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala:228]: @andrewconner the timestamp is not correct right now, but its being set or not is.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackInfoCommander.scala:228]: Cool, thanks

**Pull request merged**